### PR TITLE
CI: use autotoools for creating the tarball again

### PR DESCRIPTION
**Pull request by raveit65**

### CODE REVIEW

**Comment by raveit65** [comment]
@yetist 
Where can i add the sed command here?
`sed -i 's/$(POFILES) $(GMOFILES) \\/$(POFILES) \\/' po/Makefile.in.in`
I am missing the `build_scripts` here. Or do we have to switch to `--build scripts` first?

**Comment by yetist** [comment]
This case should switch to `--build scripts` first, then we can custom the `./autogen.sh && ./configure && make`.

**Comment by raveit65** [comment]
@yetist 
It builds but it looks like there is something missing in `build_scripts:` part.
I am missing the debug result in logs.

File: .travis.yml [hunk 1]
```diff
@@ -110,23 +110,28 @@ before_scripts:
...
   - make install
 
+build_scripts:
+  - ./autogen.sh
+  - sed -i 's/$(POFILES) $(GMOFILES) \\/$(POFILES) \\/' po/Makefile.in.in
+  - ./configure
+  - make
+
 after_scripts:
   - if [ ${BUILD_TYPE} == "autotools" ];then
```
Comment (yetist) [inline .travis.yml]: Here, now the `$BUILD_TYPE` is `scripts`. 
https://travis-ci.org/mate-desktop/mozo/jobs/549071586#L2192

**Comment by raveit65** [comment]
I think i need to switch build type in `after_scripts` part too.

File: .travis.yml (hunk 1 above)
Comment (raveit65) [reply .travis.yml]: done

**Comment by raveit65** [comment]
It seems to be working, but for some reasons i see the debug result on both commits. :confused: 
https://travis-ci.org/mate-desktop/mozo/jobs/549074720#L2260
https://travis-ci.org/mate-desktop/mozo/jobs/549074713#L2281

**Comment by raveit65** [comment]
@yetist 
Please review.

**Comment by yetist** [comment]
> It seems to be working, but for some reasons i see the debug result on both commits. 
> https://travis-ci.org/mate-desktop/mozo/jobs/549074720#L2260
> https://travis-ci.org/mate-desktop/mozo/jobs/549074713#L2281

It's ok, 106 is build on `distcheck` branch, 107 is the PR on `master` branch.

**Pull request merged**